test-app/package.json

```diff
@@ -17,60 +16,98 @@
-    "@babel/eslint-parser": "^7.17.0",
     "@ember/optional-features": "^2.0.0",
     "@ember/string": "^3.1.1",
-    "@ember/test-helpers": "^2.6.0",
-    "@embroider/test-setup": "^1.6.0",
+    "@ember/test-helpers": "^3.2.0",
+    "@embroider/test-setup": "^3.0.1",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
+    "@glint/core": "^1.0.2",
+    "@glint/environment-ember-loose": "^1.0.2",
+    "@glint/template": "^1.0.2",
+    "@tsconfig/ember": "^3.0.0",
+    "@types/ember": "^4.0.4",
+    "@types/ember__application": "^4.0.6",
+    "@types/ember__array": "^4.0.4",
+    "@types/ember__component": "^4.0.14",
+    "@types/ember__controller": "^4.0.5",
+    "@types/ember__debug": "^4.0.4",
+    "@types/ember__destroyable": "^4.0.2",
+    "@types/ember__engine": "^4.0.5",
+    "@types/ember__error": "^4.0.3",
+    "@types/ember__helper": "^4.0.2",
+    "@types/ember__modifier": "^4.0.5",
+    "@types/ember__object": "^4.0.6",
+    "@types/ember__owner": "^4.0.4",
+    "@types/ember__polyfills": "^4.0.2",
+    "@types/ember__routing": "^4.0.13",
+    "@types/ember__runloop": "^4.0.3",
+    "@types/ember__service": "^4.0.3",
+    "@types/ember__string": "^3.0.11",
+    "@types/ember__template": "^4.0.2",
+    "@types/ember__test": "^4.0.2",
+    "@types/ember__utils": "^4.0.3",
+    "@types/qunit": "^2.19.6",
+    "@types/rsvp": "^4.0.4",
+    "@typescript-eslint/eslint-plugin": "^5.62.0",
+    "@typescript-eslint/parser": "^5.62.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^2.4.1",
-    "ember-cli": "~4.3.0",
+    "concurrently": "^8.2.0",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli": "~5.2.0-beta.0",
+    "ember-cli-app-version": "^6.0.1",
     "ember-cli-babel": "^7.26.11",
-    "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-clean-css": "^2.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
+    "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-click-outside-modifier": "4.0.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-fetch": "^8.1.2",
     "ember-load-initializers": "^2.1.2",
     "ember-page-title": "^7.0.0",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.3.0",
+    "ember-qunit": "^7.0.0",
+    "ember-resolver": "^11.0.0",
+    "ember-source": "~5.2.0-beta.3",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^4.4.2",
+    "ember-template-lint": "^5.11.0",
     "ember-try": "^2.0.0",
-    "eslint": "^8.13.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-ember": "^10.6.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.2.0",
+    "eslint": "^8.44.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-ember": "^11.9.0",
+    "eslint-plugin-n": "^16.0.1",
```

Comment:
do we use `eslint-plugin-n`?

Reply:
yes, there are some Node.js files in the app, used in `test-app/.eslintrc.js`